pkg/projects/Microsoft.NETCore.App/Microsoft.NETCore.App.pkgproj

```diff
@@ -64,6 +64,11 @@
       <TargetFramework>.NETCoreApp1.1</TargetFramework>
     </Dependency>
 
+    <!-- references the host packages -->
+    <Dependency Include="Microsoft.NETCore.DotNetHostPolicy">
```

Comment:
Also, when you build the repo locally:

1) Can you confirm if the dependency being injected corresponds to the package (of HostPolicy) you built?

2) Can you confirm that the RID split package contains the hostpolicy corresponding to the package that got referenced in (1) above and does not carry hostfxr and dotnet/apphost.exe?

3) Can you confirm that sharedFrameworkPublish folder does not carry hostfxr and dotnet/apphost.exe within shared\M.N.App\<version> folder?